app/etc/pulp/server.yaml

```diff
@@ -37,10 +37,7 @@
 #
 # logging:
 #   loggers:
-#     django:
-#       level: INFO
-#     pulp.app:
-#       level: INFO
+#     '':
```

Comment:
Doesn't this also need the `'handlers': ["stream"],` part?
Also how are you validating this yaml?